acceptance/lib/helper.rb

```diff
@@ -290,8 +290,7 @@ def https_request(url, request_method, cert, key, body = nil)
   end
 
   def hup_server(host = master, timeout = 30)
-    pidfile = on(host, 'puppet master --configprint rundir').stdout.chomp + '/puppetserver' 
-    pid = on(host, "cat #{pidfile}").stdout.chomp
+    pid = on(host, "pgrep -f #{options['puppetservice']}").stdout.chomp
```

Comment:
@haus the `-f` flag is being used in this PR. So it should be resistance to such changes as long as `puppetserver` is present anywhere in the command.

Current result with `-l`:
```
# pgrep -l -f 'puppetserver'
9355 /usr/bin/java -XX:OnOutOfMemoryError=kill -9 %p -Djava.security.egd=/dev/urandom -Xms2g -Xmx2g -XX:MaxPermSize=256m -cp /opt/puppetlabs/server/apps/puppetserver/puppet-server-release.jar clojure.main -m puppetlabs.trapperkeeper.main --config /etc/puppetlabs/puppetserver/conf.d -b /etc/puppetlabs/puppetserver/bootstrap.cfg
```